product_multi_image/__manifest__.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# © 2014-2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# © 2015 Antiun Ingeniería S.L. - Jairo Llopis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Multiple Images in Products",
+    "version": "10.0.1.0.0",
+    "author": "Antiun Ingeniería, "
+              "Tecnativa, "
+              "LasLabs, "
+              "Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "website": "https://www.tecnativa.com",
```

Comment:
@FFernandez-PlanetaTIC  URL should be `https://github.com/OCA/product-attribute`

Reply:
Done, thanks!